src/stories/Blocks/material-page/material-page-skeleton.scss

```diff
@@ -0,0 +1,33 @@
+// Since we are using the Skeleton Screen Css classes connected to the existing styling
+// we deliberately not follow the BEM naming convention here.
+/* stylelint-disable plugin/stylelint-bem-namics */
+/* stylelint-disable max-nesting-depth */
```

Comment:
Consider disabling these two rules for *all* `*-skeleton.scss` files and then perhaps add something about it in our developer documentation for skeleton screens. It quickly becomes something that we just copy/paste around.